tspng/extraction.py

```diff
@@ -1,11 +1,12 @@
 #import statements
 from PIL import Image
 from tspng import MIME_TYPE
+from typing import List
 
 import json
 import os
 
-def extract_from_file(path, mime_type=MIME_TYPE):
+def extract_from_file(path: str, mime_type: str=MIME_TYPE):
```

Comment:
A type for the return value should be added. This an be done as follows:

```python
def extract_from_file(path: str, mime_type: str = MIME_TYPE) -> Dict:
```

The `Dict` class/type will need to be import from the `typing` module, `from typing import Dict, List`.

Reply:
Change made as requested